src/org/usfirst/frc/team1076/robot/subsystems/LeftRightMotors.java

```diff
@@ -15,20 +15,31 @@ public LeftRightMotors(SpeedController leftMotor, SpeedController rightMotor) {
 		this.leftMotor = leftMotor;
 		this.rightMotor = rightMotor;		
 	}
-	
+	/**
+	 * sets the speed of the left motor
```

Comment:
These look pretty good, but there are two things I would change:
- Document the expected range for `speed` (-1 to 1)
- Capitalize and punctuate all the documentation.